files/font-spoqa-han-sans/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Spoqa (spoqa.com)"
+github = "https://github.com/the1900/spoqa-han-sans"
+homepage = "http://spoqa.github.io/spoqa-han-sans/"
```

Comment:
Well I haven't thought about it.

Anyway, As you commented. I requested spoqa to add it in JSDelivr